### PR TITLE
docs/Getting_Involved/Newcomers.rst: Add a better newcomer page

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
This is only a prototype, and i need your review guys to make it perfect.
I repeat, its only a prototype, its not great yet, i just want to hear from you
what you think will make it better, and what things i need to change/add/remove.

Thanks !

### CODE REVIEW

**Comment by AbdealiJK** [comment]
ive done a quick review. others are welcome to add more comments.

**Comment by Adrianzatreanu** [comment]
Sorry; i cant see your review on my phone? Might be a problem for me or smt

**Comment by sils1297** [comment]
@AbdealiJK can't see comments either

File: docs/Getting_Involved/Newcomers.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+-------------------
+
+To get started, the first way is to meet the community. We use gitter to
+communicate, and there the helpful community will guide you.
+
+::
+
+    https://gitter.im/coala-analyzer/coala
+
+
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: don't do two newlines, one suffices

File: docs/Getting_Involved/Newcomers.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+Congratulations! You are now part of our community.
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: maybe something like the newcomer could ping a hello world to us to actually get in touch?

Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: i.e. let us know that you're there because we care

File: docs/Getting_Involved/Newcomers.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Start working
+-------------
+
+Let us know that you are willing to contribute and you will be added to our
+organization on github. This is your first step towards contributing.
+The invitation will be sent upon mail and you will have to accept it to join.
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: if you don't find the invitation, accept it at github.com/coala-analyzer

File: docs/Getting_Involved/Newcomers.rst (hunk 2 above)
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: yeah neat for the next chapter: let the newcomer ask for an invitation to the org

File: docs/Getting_Involved/Newcomers.rst [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+We use github to manage our repository. If you're not familiar with git, we
+strongly recommend following a tutorial, such as:
+
+::
+
+    https://try.github.io/levels/1/challenges/1
+
+
+If there's anything unclear, or you are encountering problems, feel free
+to contact us on git, and we will help you!
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: git(ter)?

File: docs/Getting_Involved/Newcomers.rst [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+Start working
+-------------
+
+Now is the time to pick an issue.
+
+::
+
+    https://github.com/coala-analyzer/coala/issues
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: not this link, we have a link that lists newcomer issues from all repos, it's important to always use that one

**Comment by AbdealiJK** [comment]
@Adrianzatreanu I think you force pushed after I commented.
My comments are at https://github.com/Adrianzatreanu/coala/commit/24101eee974a39d8310bbd1f3fbc5cbca878f6aa

File: docs/Getting_Involved/Newcomers.rst [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+    For coala project: https://github.com/coala-analyzer/coala/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer
+
+    Also, for our coala-bears: https://github.com/coala-analyzer/coala-bears/issues?q=is%3Aissue+is%3Aopen+label%3Adifficulty%2Fnewcomer
+
+    For more information about what bears are, please check the following link:
+    http://coala.readthedocs.org/en/latest/Users/Tutorials/Writing_Bears.html
+
+The easy issues that will help you get started are labeled as
+"difficulty/newcomer" and are only there to give you a glimpse of how it is
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: remove only, that sounds so... you know

File: docs/Getting_Involved/Newcomers.rst [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+working with us. Now pick an issue, and if you want to fix it, then leave
+a comment that you would like to get assigned. This way we don't have more
+people working on the same issue at the same time.
+Now you can start working on it.
+For more info on how to work correctly with git:
+
+::
+
+    https://github.com/coala-analyzer/coala/wiki/Getting-Started-to-Contribute#a-little-git-advice
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: actually I'd love to remove this page completely if this is done

File: docs/Getting_Involved/Newcomers.rst [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+
+.. note::
+
+    Careful! For more information about creating commit messages, check:
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: that sounds like a warning or something dangerous, can we make it a bit friendlier?

File: docs/Getting_Involved/Newcomers.rst [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+Sending your changes
+--------------------
+
+Now that you've fixed the issue, you've tested it and you think it is ready
+to be used, create a commit and push it to your fork, using:
+
+::
+
+    "$ git push origin fix-branch -f", where -f is short for --force.
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: you should use double backticks for inline code snippets

Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: omit the `-f` at initial push don't confuse with that yet IMO

File: docs/Getting_Involved/Newcomers.rst [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+Congratulations! You have now created your first Pull Request!
+Now your code will be tested by our checkers and also gitmate, which will tell
+you stuff as if your commit message was wrongly written. If you have passed
+all those tests, your code has to be reviewed by one of your maintaining
+developers. If they consider that you have issues on your code, you will be
+able to modify it and then add the files manually again using git, and use
+git commit --amend. This will edit your last commit. This command also allows
+you to edit your commit message.
+The other way around, if it gets merged, you are good to go! You have
+contributed to coala! Congratulations!
```
Comment (sils1297) [inline docs/Getting_Involved/Newcomers.rst]: I think this deserves a bit more love and another section maybe?

**Pull request closed without merging**